RestApi fails on task creation

**CloudNet v4**
**Description of the bug:**

Try to create a task with the restapi fails into a parsing error.


**Steps to reproduce this bug:**

1. Create a object like this:
```dart
ServiceTask(properties: null, includes: null, templates: null, deployments: null, processConfiguration: ProcessConfiguration(environment: MINECRAFT_SERVER, maxHeapMemorySize: 512, jvmOptions: null, processParameters: null), name: hello, minServiceCount: 0, startPort: 44955, runtime: null, javaCommand: java, disableIpRewrite: null, maintenance: false, autoDeleteOnStop: true, staticServices: false, associatedNodes: null, groups: null, deletedFilesAfterStop: null)
```
2. Send this as a Json body to this endpoint:
```
http://YOUR_CLOUDNET_NODE:2812/api/v2/task
```

**Expected behavior:**

Everything is ok and i got a 200 response from the restapi!


**Screenshots:**

![image](https://user-images.githubusercontent.com/6745190/147385099-f58b43a9-cd45-45e1-a52d-3e80d28515ff.png)


**Environment and used hardware:**

System specifications:  Apple M1 16Gb memory
OS: OS X 12.0.1
Version: CloudNet v4.0.0


**Additional context (optional):**


```
[25.12 13:36:38.217] SEVERE: Exception posting http request to handler de.dytanic.cloudnet.ext.rest.v2.V2HttpHandlerTasks
java.lang.IllegalArgumentException: Unable to create instance of de.dytanic.cloudnet.driver.service.ProcessConfiguration with arguments [MINECRAFT_SERVER, 512, null, null]
	at de.dytanic.cloudnet.common.document.gson.RecordTypeAdapterFactory$RecordTypeAdapter.read(RecordTypeAdapterFactory.java:141)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$1.read(ReflectiveTypeAdapterFactory.java:131)
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:222)
	at com.google.gson.Gson.fromJson(Gson.java:963)
	at com.google.gson.Gson.fromJson(Gson.java:1034)
	at com.google.gson.Gson.fromJson(Gson.java:1006)
	at de.dytanic.cloudnet.common.document.gson.JsonDocument.toInstanceOf(JsonDocument.java:156)
	at de.dytanic.cloudnet.ext.rest.v2.V2HttpHandlerTasks.handleTaskCreateRequest(V2HttpHandlerTasks.java:87)
	at de.dytanic.cloudnet.ext.rest.v2.V2HttpHandlerTasks.handleBearerAuthorized(V2HttpHandlerTasks.java:44)
	at de.dytanic.cloudnet.http.V2HttpHandler.handle(V2HttpHandler.java:84)
	at de.dytanic.cloudnet.driver.network.netty.http.NettyHttpServerHandler.handleMessage0(NettyHttpServerHandler.java:204)
	at de.dytanic.cloudnet.driver.network.netty.http.NettyHttpServerHandler.handleMessage(NettyHttpServerHandler.java:116)
	at de.dytanic.cloudnet.driver.network.netty.http.NettyHttpServerHandler.channelRead0(NettyHttpServerHandler.java:92)
	at de.dytanic.cloudnet.driver.network.netty.http.NettyHttpServerHandler.channelRead0(NettyHttpServerHandler.java:45)
	at io.netty.channel.SimpleChannelInboundHandler.channelRead(SimpleChannelInboundHandler.java:99)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:324)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:296)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:166)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:722)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:658)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:584)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:496)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
	at de.dytanic.cloudnet.common.document.gson.RecordTypeAdapterFactory$RecordTypeAdapter.read(RecordTypeAdapterFactory.java:139)
	... 39 more
Caused by: java.lang.NullPointerException: jvmOptions is marked non-null but is null
	at de.dytanic.cloudnet.driver.service.ProcessConfiguration.<init>(ProcessConfiguration.java:29)
	... 45 more
```

--- a/driver/src/main/java/eu/cloudnetservice/cloudnet/driver/network/netty/http/NettyHttpServerHandler.java
+++ b/driver/src/main/java/eu/cloudnetservice/cloudnet/driver/network/netty/http/NettyHttpServerHandler.java
@@ -275,10 +275,13 @@ final class NettyHttpServerHandler extends SimpleChannelInboundHandler<HttpReque
       httpHandlerEntry.httpHandler().handle(fullPath.toLowerCase(), context);
       return true;
     } catch (Exception exception) {
-      LOGGER.severe(
+      LOGGER.finer(
         "Exception posting http request to handler %s",
         exception,
         httpHandlerEntry.httpHandler().getClass().getName());
+      // assume that the request was bad so that the handler was unable to handle it
+      context.response().status(HttpResponseCode.BAD_REQUEST);
+      // continue with the next handler in the chain
       return false;
     }
   }

